airbyte-integrations/connectors/destination-weaviate/destination_weaviate/client.py

```diff
@@ -112,23 +148,76 @@ def flush(self, retries: int = 3):
 
         self.buffered_objects.clear()
 
-    def delete_stream_entries(self, stream_name: str):
-        class_name = stream_to_class_name(stream_name)
+    def delete_stream_entries(self, stream:AirbyteStream):
+
+        class_name = stream_to_class_name(stream.name)
         try:
-            original_schema = self.client.schema.get(class_name=class_name)
```

Comment:
the reason this was done this way was to preserve things like moduleconfig which you can not get from the Airbyte stream. It's important to preserve the original schema as is to delete the stream entries.

Reply:
Oh, is this module config with respect to the original Weaviate's potentially custom configuration, which is not 'inferable' via just introspecting air byte properties?

Does it make sense to have the module config floated into the Config for the air byte destination? This would seem like a natural place for the module config, but I am not familiar enough with the nuances to know of the pitfalls. 

Thanks for looking over this.